docs/docs/react-test-renderer.md

```diff
@@ -0,0 +1,213 @@
+---
+id: react-test-renderer
+title: React Test Renderer
+permalink: docs/react-test-renderer.html
+layout: docs
+category: Reference
+---
+
+**Importing**
+
+```javascript
+import ReactTestRenderer from 'react-test-renderer'; // ES6
+const ReactTestRenderer = require('react-test-renderer'); // ES5 with npm
+```
+
+## Overview
+
+This package provides an experimental React renderer that can be used to render React components to pure JavaScript objects, without depending on the DOM or a native mobile environment.
+
+Essentially, this package makes it easy to grab a snapshot of the "DOM tree" rendered by a React DOM or React Native component without using a browser or jsdom.
+
+Example:
+
+```javascript
+const ReactTestRenderer = require('react-test-renderer');
+
+const testRenderer = ReactTestRenderer.create(
+  <Link page="https://www.facebook.com/">Facebook</Link>
+);
+
+console.log(testRenderer.toJSON());
+// { type: 'a',
+//   props: { href: 'https://www.facebook.com/' },
+//   children: [ 'Facebook' ] }
+```
+
+You can use Jest's snapshot testing feature to automatically save a copy of the JSON tree to a file and check in your tests that it hasn't changed: http://facebook.github.io/jest/blog/2016/07/27/jest-14.html.
+
+You can traverse a ReactElement by methods ReactTestInstance provides.
```

Comment:
Let's change to "You can also traverse the output to find specific nodes and make assertions about them."